samples: cellular: location: Enable Wi-Fi by default for Thingy91x (#18719)
nrfconnect/sdk-nrf: samples/cellular/location/README.rst

See :ref:`cmake_options` for more instructions on how to add these options.

.. note::

>    Thingy:91 X build has Wi-Fi support enabled by default. You do not need to add any overlays.

Reviewer: ```suggestion
   The Thingy:91 X build supports Wi-Fi by default.
   You do not need to add any overlays.
```
